test_general/api/test_app.py

```diff
@@ -666,3 +646,41 @@ def test_chunking_strategy_additional_params():
         response_multipage_true_combine_chars_5000.json()
         != response_from_multipage_false_combine_chars_0.json()
     )
+
+
+def test_encrypted_pdf():
+    """
+    Test that we throw an error if a pdf is password protected.
+    A pdf can be encrypted but still readable - don't throw an error here.
+    """
+    client = TestClient(app)
+    test_file = Path("sample-docs") / "layout-parser-paper-fast.pdf"
+    original_pdf = PdfReader(test_file)
+    temp_file = tempfile.NamedTemporaryFile()
```

Comment:
some nit: we should use temp file with `with tempfile.NamedTemporaryFile() as temp_file:` so that we close the temp file